src/handler.lua

```diff
@@ -34,6 +35,26 @@ local JwtKeycloakHandler = {
     PRIORITY = priority
 }
 
+-------------------------------------------------------------------------------
+-- custom helper function of the extended plugin "jwt-keycloak"
+-- --> extract allowed algorithms from schema
+-------------------------------------------------------------------------------
+local function get_allowed_algorithms_from_schema()
```

Comment:
would a constant here also be ok and more performance wihtout having to nested loops to iterate over schema?